pydantic_ai_slim/pydantic_ai/ag_ui.py

```diff
@@ -378,6 +398,15 @@ async def run_ag_ui(
                 run_id=run_input.run_id,
             ),
         )
+    finally:
```

Comment:
If we're also going to call this on failure, I think we should pass the exception to the callback handler.

I'm also fine with just calling this after the `async for event` is complete.

Reply:
> I'm also fine with just calling this after the async for event is complete.

Make sense.  Updated to only run on success